indico/modules/rb_new/client/js/modules/calendar/reducers.js

```diff
@@ -39,6 +40,23 @@ export default combineReducers({
                 return {...state, rows: [], date: action.date};
             case calendarActions.ROWS_RECEIVED:
                 return {...state, rows: action.data};
+            case bookRoomActions.CREATE_BOOKING_SUCCESS: {
+                const {data: {occurrences, room_id: roomId}} = action;
+                const {rows} = state;
+                let newRows = [...rows];
```

Comment:
you overwrite this two lines below, so you can just do `const newRows = rows.map(...);`